src/journeys/enterEmailService.js

```diff
@@ -0,0 +1,28 @@
+function validateEnterEmail(req, res) {
+  try {
+    const { enterEmail } = req.body;
+    const queryParams = req.query;
+    const editMode = queryParams.editMode;
+
+    if (enterEmail && enterEmail !== "") {
+      // Check if in edit mode and redirect accordingly
+      if (editMode) {
```

Comment:
I don't think this if is necessary because both situations have the same output.

Reply:
That's true , will remove. Thank you